src/jobs/test.yml

```diff
@@ -0,0 +1,47 @@
+parameters:
+  before-setup:
+    description: steps to execute before setup
+    type: steps
+    default: []
+  cache_key_prefix:
+    description: key of cache
+    type: string
+    default: ''
+  executor:
+    description: executor for test
+    type: executor
+  plugin_name:
+    description: name of Redmine plugin
+    type: string
+    default: ''
+  script:
+    description: steps to execute the test
+    type: steps
+    default:
+      - run:
+          bin/rails redmine:plugins:test NAME=$CIRCLE_PROJECT_REPONAME
```

Comment:
How about assuming a plugin name as `Install plugin` does? Is this too implicit?

```sh
- run:
    if [ -n "<< parameters.plugin_name >>" ]; then
      bin/rails redmine:plugins:test NAME=<< parameters.plugin_name >>
    else
      bin/rails redmine:plugins:test NAME=$CIRCLE_PROJECT_REPONAME
    fi
```

Reply:
Good catch! thanks, I update the default test script 6f2c5fe

Reply:
something wrong, I will fix it.
https://circleci.com/gh/hanachin/orb_works/86

Reply:
I realize we can not use other parameter in default value of the parameter.
I need to find another way :mag:

Reply:
Fix in 0a4accc and eb2f622

@fkdkent Could you please take a look?

I tested the `dev:eb2f622` in 3 different config

plugin name assume from repo name
https://github.com/hanachin/orb_works/compare/test_plugin_name_assume_from_repo_name
https://circleci.com/workflow-run/7e5b43c0-505e-4c41-b9af-9288787393e1

set wrong plugin name
https://github.com/hanachin/orb_works/compare/master...test_plugin_name_parameter_failed
https://circleci.com/workflow-run/75dddfe0-e564-4e69-96bd-710a8d4fb7ca

set correct plugin name
https://github.com/hanachin/orb_works/compare/master...test_plugin_name_parameter_success
https://circleci.com/workflow-run/37041483-40d3-426e-963f-1066d1e0a07e

Reply:
CI on `test_plugin_name_parameter_failed` branch fails as expected, and the others succeed as expected.

LGTM 👍